configs/defs/cassandra-true.textproto

```diff
@@ -0,0 +1,101 @@
+# Some rules need a workaround because the configuration value is not on the same line of the key.
+# Therefore, a workaround is needed. Using the \0 delimiter does not seem to execute the regex on all file content
+#
+# The approach used in the following rules set the file content split delimiter to "true" and checks with a regexp
+# if the required configuration value is enabled or not. The regexp was thoroughly tested to avoid false positives.
+
+benchmark_configs: {
+  id: "cassandra-auditing"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/a:apache:cassandra:3.11" version: "1.0.0" benchmark_document: "CIS Apache Cassandra 3.11" }
+    title: "Ensure that auditing is enabled"
+    description:
+      "Audit logging in Cassandra logs every incoming CQL command request, Authentication "
+      "(successful as well as unsuccessful login) to C* node. Currently, there are two "
+      "implementations provided, the custom logger can be implemented and injected with the "
+      "class name as a parameter in cassandra.yaml."
+    rationale:
+      "Unauthorized attempts to create, drop or alter users or data should be a concern."
+    remediation:
+      "Enable the Cassandra Auditing feature. Note that on the Open Source version this feature is available only starting from version 4.\n"
+      "Open the dse.yaml file in a text editor\n"
+      "In the audit_logging_options section, set enabled to true.\n"
+      "```# Audit logging options\n"
+      "audit_logging_options:\n"
+      "enabled: true```\n"
+      "You must also define where you want logging to go, add either of the following lines:\n"
+      "Set the logger option to either CassandraAuditWriter, which logs to a table, or SLF4JAuditWriter, which logs to the SLF4J logger."
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+    "check_alternatives:{"
```

Comment:
Note that in the def files, these are [BenchmarkScanInstructionDef](https://github.com/google/localtoast/blob/4086becfcff0f5669edab02d4f262f35300c3b3a/scannerlib/proto/scan_instructions.proto#L25) proto messages, not BenchmarkScanInstruction. You can just add the `generic:{}` wrapper around the instructions to fix this. See the [cos.textproto](https://github.com/google/localtoast/blob/4086becfcff0f5669edab02d4f262f35300c3b3a/configs/defs/cos.textproto#L119) file for an example.